extension/lib/shopgate/pipeline/CallFailedError.js

```diff
@@ -0,0 +1,12 @@
+class ShopgatePipelineCallFailedError extends Error {
+  constructor (previous) {
+    super('Failed processing pipeline request.')
+    this._previous = previous
+  }
+
+  get previous () {
```

Comment:
can there be type hint Error or will it lead to wrong typing?

Reply:
I've missed it, done.